packages/astro/src/@types/astro.ts

```diff
@@ -294,8 +295,23 @@ export interface AstroUserConfig {
 		/**
 		 * @docs
 		 * @name devOptions.hostname
+		 * @type {string | boolean}
+		 * @default `false`
+		 * @version 0.24.0
+		 * @description
+		 * Set which network IP addresses the dev server should listen on (i.e. 	non-localhost IPs).
+		 * - `false` - do not expose on a network IP address
+		 * - `true` - listen on all addresses, including LAN and public addresses
+		 * - `[custom-address]` - expose on a network IP address at `[custom-address]`
+		 */
+		host?: string | boolean;
+
+		/**
+		 * @docs
+		 * @name devOptions.host
 		 * @type {string}
-		 * @default `'localhost'`
+		 * @default `localhost`
+		 * @deprecated Use --host instead
```

Comment:
how should we handle this in the docgen script? 

I think my suggestion would be to ignoring it completely, and add a note in the `host` `@description` body that it used to be called `hostname`.

Reply:
Made that refactor. Waiting for the docgen updates to merge, but will verify my `> blockquote` looks right once those are in 👍